### PR TITLE
style_appmenu_button_in_headerbar

### PR DESCRIPTION
**Pull request by Feichtmeier**
peek 2018-05-08 16-29
Closes https://github.com/ubuntu/gtk-communitheme/issues/413

### CODE REVIEW

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/gtk-communitheme-pr418`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/gtk-communitheme-pr418` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

File: Communitheme/gtk-3.0/_common.scss [hunk 1]
```diff
@@ -4550,12 +4550,25 @@ button.titlebutton {
...
   padding: 2px;
 
-  &:not(.close) {
+  &:not(.close):not(.appmenu){
     &:hover {
       @include draw_circle($c) ;
       &:backdrop:hover{@include draw_circle(darken($headerbar_fg_color,50%), $close:true)}
     }
   }
+  &.appmenu{
```
Comment (clobrano) [inline Communitheme/gtk-3.0/_common.scss]: I think you can replace this adding "button.titlebutton.appmenu" at row 1586

    - button, button.flat
    + button, button.flat, button.titlebutton.appmenu

Comment (Feichtmeier) [reply Communitheme/gtk-3.0/_common.scss]: Ty

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/gtk-communitheme-pr418`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/gtk-communitheme-pr418` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by Feichtmeier** [comment]
@clobrano changes done. Please review so I can merge

**Comment by clobrano** [comment]
LGTM :+1:

**Pull request merged**